jobs/base.bull.job.js

```diff
@@ -77,15 +77,21 @@ class BaseJob {
 	startWorker() {
 		this.queue.process(this.concurrency, async (job, done) => {
 			const apmTransaction = apm.startTransaction(this.constructor.name, 'job');
+			const startTime = Date.now();
+
+			this.logger = this.logger.child({ jobId: job.id, id: (job.data && job.data.id) });
+			if (this.logger.debug()) {
+				this.logger = this.logger.child({ data: job.data });
+			}
+
 			const doneWrapper = (err) => {
 				apmTransaction.result = err ? 'error' : 'success'
 				apmTransaction.end();
+				const executionTime = Date.now() - startTime
+				this.logger.debug(`${this.constructor.name} completed in ${executionTime} milliseconds`, { executionTime, attempt: job.attemptsMade, job: this.constructor.name })
```

Comment:
`jobName` should already be getting logged, so i don't think you need the `job` field here